Prep docs for release of portal wide metadata (#320)
AlexsLemonade/scpca-docs: docs/CHANGELOG.md

<!-- PUT THE NEW CHANGELOG ENTRY RIGHT BELOW THIS -->
<!-------------------------------------------------->

## PLACEHOLDER FOR PORTAL WIDE METADATA

> * Metadata for all samples from all projects on the Portal can now be downloaded as one tab-separated values file.

Reviewer: gratuitous wording suggestion
```suggestion
* Metadata for all samples from all projects on the Portal can now be downloaded in a single tab-separated values file.
```
